core/shared/src/main/scala-2/zio/internal/macros/LayerMacroUtils.scala

```diff
@@ -120,6 +120,8 @@ private[zio] trait LayerMacroUtils {
       method = provideMethod,
       exprToNode = getNode,
       typeToNode = typeToNode,
+      combineTypes = (_, _) => ???,
+      expandRIn = (_, _) => ???,
```

Comment:
I'm guessing these should never be called, but could we throw a more meaningful error in such case? Ideally via `report.errorAndAbort` (although not sure if that's possible)

Reply:
Fixed